lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -434,10 +434,6 @@ def draw(self, renderer):
 
         # add the projection matrix to the renderer
         self.M = self.get_proj()
-        renderer.M = self.M
-        renderer.vvec = self.vvec
-        renderer.eye = self.eye
-        renderer.get_axis_position = self.get_axis_position
```

Comment:
This convention of hanging `M` and other things off of the renderer object has existed since before I started dealing with mplot3d 10 years ago. I don't particularly know the reason for it (possibly due to the oddities of the axes3d design back then that I eventually smoothed over). I agree it probably isn't needed anymore, but given how long it has been there, I am hesitant about ripping them out without a deprecation notice.

Reply:
This is a bit of a pain, as adding a property (in order to use the deprecations) is not possible on an instance, but I think I managed to do this.